stingray/powerspectrum.py

```diff
@@ -504,3 +513,48 @@ def trace_maximum(self, min_freq=None, max_freq=None):
             max_positions.append(np.where(ps.power == max_power)[0][0])
 
         return np.array(max_positions)
+
+    def rebin_time(self, dt_new, method='sum'):
+
+        """
+        Rebin the Dynamic Power Spectrum to a new time resolution. While the new
+        resolution need not be an integer multiple of the previous time
+        resolution, be aware that if it is not, the last bin will be cut
+        off by the fraction left over by the integer division.
+
+        Parameters
+        ----------
+        dt_new: float
+            The new time resolution of the Dynamica Power Spectrum. Must be larger than
+            the time resolution of the old Dynamical Power Spectrum!
+
+        method: {"sum" | "mean" | "average"}, optional, default "sum"
+            This keyword argument sets whether the counts in the new bins
+            should be summed or averaged.
+
+
+        Returns
+        -------
+        time_new: numpy.ndarray
+            Time axis with new rebinned time resolution.
+
+        dynspec_new: numpy.ndarray
+            New rebinned Dynamical Power Spectrum.
+        """
+        
+        if dt_new < self.dt:
+            raise ValueError("New time resolution must be larger than "
+                             "old time resolution!")
+
+        dynspec_new = []
+        for data in self.dyn_ps:
+            time_new, bin_counts, bin_err, _ = \
+                utils.rebin_data(self.time, data, dt_new,
+                                 yerr=self.lc.counts_err, method=method)
+            dynspec_new.append(bin_counts)
+
+        self.time = time_new
+        self.dyn_ps = np.array(dynspec_new)
+        self.dt = dt_new
+
+        return time_new, dynspec_new
```

Comment:
@evandromr Yeah, I thought of it and decided to return. I agree it is not necessary. I think I'll remove them.